src/test/java/net/minecraftforge/debug/world/BiomeModifierTest.java

```diff
@@ -136,31 +170,27 @@ public void run(final CachedOutput cache) throws IOException
             {
                 PlacedFeature.DIRECT_CODEC.encodeStart(ops, feature)
                     .resultOrPartial(msg -> LOGGER.error("Failed to encode {}: {}", featurePathString, msg)) // Log error on encode failure.
-                    .ifPresent(json -> // Output to file on encode success.
-                    {
-                        try
-                        {
-                            DataProvider.saveStable(cache, json, featurePath);
-                        }
-                        catch (IOException e) // The throws can't deal with this exception, because we're inside the ifPresent.
-                        {
-                            LOGGER.error("Failed to save " + featurePathString, e);
-                        }
-                    });
+                    .ifPresent(LamdbaExceptionUtils.rethrowConsumer(json -> DataProvider.saveStable(cache, json, featurePath))); // Output to file on encode success.
+
+                BiomeModifier.DIRECT_CODEC.encodeStart(ops, addFeature)
+                    .resultOrPartial(msg -> LOGGER.error("Failed to encode {}: {}", addFeaturePathString, msg)) // Log error on encode failure.
+                    .ifPresent(LamdbaExceptionUtils.rethrowConsumer(json -> DataProvider.saveStable(cache, json, addFeaturePath)));
```

Comment:
And then especially with the other suggestion, you could basically have a helper method that takes: `ops, cache, directory, name, modifier` and have the helper method do the `directory.resolve(name + ".json")` for what it saves it to.